applications/RomApplication/rom_application_variables.cpp

```diff
@@ -20,4 +20,16 @@ namespace Kratos
     KRATOS_CREATE_VARIABLE( Matrix, ROM_LEFT_BASIS )
     KRATOS_CREATE_VARIABLE( double, HROM_WEIGHT )
     KRATOS_CREATE_VARIABLE( Vector, ROM_SOLUTION_INCREMENT )
+    KRATOS_CREATE_VARIABLE( Vector, ROM_SOLUTION_BASE )
+    KRATOS_CREATE_VARIABLE( Vector, ROM_SOLUTION_TOTAL )
+    KRATOS_CREATE_VARIABLE( Vector, SOLUTION_BASE )
+    KRATOS_CREATE_VARIABLE( vector<Matrix>, SVD_PHI_MATRICES )
+    KRATOS_CREATE_VARIABLE( vector<Matrix>, NN_LAYERS )
+    KRATOS_CREATE_VARIABLE( Vector, SOLUTION_REFERENCE )
+    KRATOS_CREATE_VARIABLE( bool, UPDATE_PHI_EFFECTIVE_BOOL )
+    
+    // KRATOS_CREATE_VARIABLE( vector<EigenDynamicMatrix>, SVD_PHI_MATRICES )

```

Comment:
Cleanup, unless required for something. Is it simply a leftover?

Reply:
Sure, cleaning it up